src/plugins/vis_type_timelion/public/components/bar_series/index.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { BarSeries, ScaleType } from '@elastic/charts';
+
+interface BarSeriesComponentProps {
```

Comment:
`BarSeriesComponentProps` and `AreaSeriesComponentProps` have similar props. Also you use their from one place. Maybe it make sense to create one interface for both

Reply:
Removed. We have only 2 props, I think we can avoid interface